icons/f-secure.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" aria-labelledby="simpleicons-fsecure-icon" role="img" viewBox="0 0 24 24"><title id="simpleicons-fsecure-icon">F-Secure icon</title><path d="M23.7 2.2a35.05 35.05 0 0 0-22.43-.45l.25-.07.26-.07a2.2 2.2 0 0 0-1.6 2.76v-.02c.4 1.6.97 3.42 1.65 5.09.01.04.04.02.03-.02-.08-.78.5-1.76 1.67-2.13a26.75 26.75 0 0 1 17.51.36l-.18-.06-.2-.06c.85.3 1.8-.17 2.08-1.04.36-1.12.72-2.5.99-3.93l.04-.23c0-.06-.05-.1-.07-.12zm0 0"/><path d="M6.64 7.16c-.91.2-2.2.5-3.12.82-1.9.65-1.82 3-.8 3.75.07-.44.52-1.03 1.04-1.23 1.74-.7 3.78-1.2 5.9-1.38h.08a10.97 10.97 0 0 1-3.1-1.98v.01h.02zm0 0"/><path d="M21.42 11.48a22.24 22.24 0 0 0-17.84-.6l.15-.05.15-.05c-.88.35-1.46 1.46-.88 2.5a40.25 40.25 0 0 0 3.41 5.29l-.07-.1c-.19-.63.14-1.8 1.36-2.18 3.33-1.04 6.87-.6 9.22.36.65.26 1.6.1 2.12-.7.79-1.2 1.6-2.6 2.32-4.06l.1-.25c.04-.07 0-.13-.04-.16zm0 0"/><path d="M15.96 19.84a10.35 10.35 0 0 1-2.66-1.93c-.38-.37-.9-.9-1.48-1.52-1.27 0-2.52.13-3.85.55-1.22.4-1.34 1.85-.8 2.6.92 1.14 1.55 1.76 2.58 2.84a2.89 2.89 0 0 0 4.07.03c.8-.82 1.29-1.36 2.2-2.4.04-.06.02-.14-.06-.17zm0 0"/></svg>
```

Comment:
Your latest version is not centered correctly, also it seems that over the changes your SVG has degraded a bit 😞 I went ahead and update the icon for you (I hope you don't mind):

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" aria-labelledby="simpleicons-fsecure-icon" role="img" viewBox="0 0 24 24"><title id="simpleicons-fsecure-icon">F-Secure icon</title><path d="M23.93 2.94a35.94 35.94 0 0 0-22.24-.6A2.22 2.22 0 0 0 .07 5.09c.4 1.6.98 3.44 1.68 5.11.01.04.03.02.03-.02-.1-.78.5-1.77 1.68-2.13a27.56 27.56 0 0 1 17.39.23c.86.3 1.82-.17 2.1-1.06.7-2.25.98-3.78 1.05-4.16.01-.07-.05-.1-.07-.12z"/><path d="M6.65 7.89c-.86.18-2.05.46-2.94.76-1.78.61-1.7 2.78-.75 3.47.07-.4.5-.95.98-1.13 1.78-.7 3.69-1.12 5.62-1.29-.98-.4-1.94-.97-2.9-1.81"/><path d="M20.83 12.23a21.16 21.16 0 0 0-16.45-.65c-.85.32-1.38 1.35-.85 2.33a38.16 38.16 0 0 0 3.15 4.8c-.17-.58.13-1.66 1.27-2.01 3.15-.97 6.46-.56 8.66.33.62.25 1.5.1 1.99-.64a38.62 38.62 0 0 0 2.29-4.02c.03-.06 0-.11-.06-.14"/><path d="M15.72 20a9.92 9.92 0 0 1-2.5-1.8c-.34-.34-.84-.83-1.37-1.41-1.2 0-2.37.12-3.62.52-1.16.36-1.27 1.7-.76 2.4.86 1.07 1.46 1.65 2.42 2.64a2.74 2.74 0 0 0 3.82.02c.75-.75 1.21-1.25 2.06-2.21.05-.05.03-.14-.05-.16"/></svg>
```
![difference preview](https://user-images.githubusercontent.com/3742559/47344026-1fbf0280-d6b0-11e8-9940-e532c3ed519b.gif)